priv/dqe_idx_pg.schema

```diff
@@ -1,10 +1,23 @@
 %% -*- erlang -*-
 
-%% @doc The TCP endpoint for the backend server.
+%% @doc The TCP endpoint for the backend server. Doesn't support hostnames.
+%% This setting overwrites backend_host and backend_port connections.
 {mapping, "idx.pg.backend", "dqe_idx_pg.server",
  [{default, {"127.0.0.1", 5432}},
```

Comment:
Sorry for only catching this now didn't saw it earlier, as long as the backend has a default it **always** will be set to backend_host / backend_port can't ever take effect.